sotodlib/site_pipeline/ml_mapmaking.py

```diff
@@ -0,0 +1,312 @@
+from argparse import ArgumentParser
+import numpy as np, sys, time, warnings, os, so3g
+from sotodlib.core import Context, AxisManager, IndexAxis
+from sotodlib import mapmaking
+from sotodlib.io import metadata   # PerDetectorHdf5 work-around
+from sotodlib import tod_ops
+from sotodlib.tod_ops import filters
+from pixell import enmap, utils, fft, bunch, wcsutils, mpi
+#import moby2.analysis.socompat
```

Comment:
You can just delete this line.  It was in here so that context system would be able to load ACT data properly, but that has a different work-around now (should the need arise).